src/analyzer/protocol/sip/sip-analyzer.pac

```diff
@@ -16,8 +20,16 @@ refine flow SIP_Flow += {
 		return content_length;
 		%}
 
-	function proc_sip_request(method: bytestring, uri: bytestring, vers: SIP_Version): bool
+	function proc_sip_request(junk: bytestring, method: bytestring, uri: bytestring, vers: SIP_Version): bool
 		%{
+		if ( junk.length() )
+			{
+			zeek::String zs{junk.begin(), junk.length(), false};
+			auto addl = zs.Render();
```

Comment:
just as a note, seems we should add a `Render()` version returning a string sometime to avoid the manual memory mgmt.